src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/fs/TeeStep.java

```diff
@@ -63,6 +62,20 @@ public StepExecution start(StepContext context) throws Exception {
         return new Execution(context, file);
     }
 
+    private static final class TeeTail extends BodyExecutionCallback.TailCall {
```

Comment:
Needs a `serialVersionUID`.

Reply:
Apparently, since I lack write permission in this repository, I cannot mark _my own_ review comments as resolved! @github surely that is not by design?